### ISSUE TITLE
Suggestion to simplify Dockerfile

### ISSUE THREAD

**Issue by chgl** [swade1987/deprek8ion#13]
Thank you for your work, this is a really useful tool!

I have a minor suggestion to simplify the Dockerfile:

```Dockerfile
FROM instrumenta/conftest:v0.18.2
LABEL MAINTAINER Steven Wade <steven@stevenwade.co.uk>
COPY policies/* /policies/
ENTRYPOINT ["/conftest", "test", "-p", "/policies"]

# Usage: docker run --rm -v $PWD/demo:/demo deprek8ion:latest /demo
```

This would make the install.sh unnecessary. I can create a PR if you're interested. 

One more thing: would it be possible to also host the container on docker hub - it has a very straight-forward GitHub integration and isn't down currently...

**Comment by swade1987** [comment]
@chgl a PR for this would be much appreciated.
I am planning to move this over to gcr.io in the coming week.

**Comment by swade1987** [comment]
Closed as part of #14
